doc/configuration.rst

```diff
@@ -266,6 +266,14 @@ worker-wait-jitter
   workers do not ask the scheduler for another job at the same time.
   Default: 5.0
 
+worker-install-shutdown-handler
+  If true, workers will stop requesting new work and finish running pending
+  tasks after receiving a `SIGUSR1` signal. This provides a hook for
+  gracefully shutting down workers that are in the process of running
+  (potentially expensive) tasks. Note this hook does not work on Windows
+  operating systems, or with jobs launched outside the main execution thread.
+  Defaults to true.
+
```

Comment:
Actually call it "no_install_shutdown_handler". "no" because False should be default for all bool parameters and underscores because that's the new convention. (docs lag behind a lot, sorry about that!)